src/main/java/org/desarrolladorslp/technovation/services/impl/EventServiceImpl.java

```diff
@@ -0,0 +1,68 @@
+package org.desarrolladorslp.technovation.services.impl;
+
+import org.desarrolladorslp.technovation.config.auth.TokenInfo;
+import org.desarrolladorslp.technovation.config.auth.TokenInfoService;
+import org.desarrolladorslp.technovation.controller.dto.EventDTO;
+import org.desarrolladorslp.technovation.models.Session;
+import org.desarrolladorslp.technovation.repository.SessionRepository;
+import org.desarrolladorslp.technovation.services.EventService;
+import org.desarrolladorslp.technovation.services.SessionService;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.PropertyMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Service
+public class EventServiceImpl implements EventService {
+
+    private SessionRepository sessionRepository;
+
+    private ModelMapper modelMapper;
+
+    @Override
+    public List<EventDTO> listEvents(int year, int month, UUID userId) {
+
+        if (month < 1 || month > 12) {
+            throw new IllegalArgumentException("Invalid Month");
+        }
+
+        List<Session> sessions = sessionRepository.getSessionsByUser(userId, year, month);
+        return sessions.stream().map(this::convertToDTO).collect(Collectors.toList());
+    }
```

Comment:
I think it was a good decision to move DTOs creation to services. What do you think?